4_Methodology/Public_review.adoc

```diff
@@ -1,57 +1,148 @@
+|30 days
+|iTC has voted according to Terms of Reference to propose this as the final document. Public (i.e. from non-iTC participants) comments are accepted during this period
 
-Each comment must have a suggested resolution be proposed if a change is needed to the document.
+|Proposed Update
+|10 days
+|iTC reviews any further comments and prepares the document for final publishing (updating all dates, producing official versions for publication)
 
-The public review period is scheduled for 45 days from <some date>, ending on <some later date>. 
+|Final Document Published
+|
+|Documents are posted to Common Criteria Portal
 
-=== GitHub Review Process
-To use GitHub to submit comments, you must have a GitHub account (and it is assumed you know how to use GitHub). Each comment should be submitted as an individual [Issue](https://github.com/biometricITC/cPP-biometrics/issues) with the Label "Public Review" assigned. Pull Requests created for any issues will be linked to for tracability.
+|===
+
+The iTC may decide, based on the comments received during the Public Review Draft 1 period, that a Public Review Draft 2 period is needed. Public announcement of a second review draft or a proposed draft will be made once all comments have been addressed.
+
+=== Review Process
+There are two ways to contribute comments and suggestions to the iTC. The first is through GitHub, the second by spreadsheet. It should be noted however that comments that are received via the spreadsheet will be added to the github platform to allow for a comprehensive discussion. Also, feedback for comments is only provided via the answers in the github comments. 
 
-=== Spreadsheet Review Process
+Each comment should have a suggested resolution be proposed if a change is needed to the document.
+
+==== GitHub Review Process
+To use GitHub to submit comments, you must have a GitHub account (and it is assumed you know how to use GitHub). Each comment should be submitted as an individual https://github.com/biometricITC/cPP-biometrics/issues[Issue] with the Label "Public Review" assigned. Pull Requests created for any issues will be linked to these Issues for tracability.
+
+==== Spreadsheet Review Process
 Along with the link in <<DocTable>> for each document is a link to the spreadsheet to be used for comments. Email the spreadsheets to <some email address>.
 
+== Additional Notes
+The documents under review are the current state of output. Two major areas of discussion within the iTC that are still under consideration within are noted below. 
+
+Comments on these specific topics can be made separately.
+
+=== Publish as PP-Module or PP
+The original focus for the iTC was to create a standalone PP that would be evaluated separately that defines security requirements to provide the secure environment for the biometric enrolment and verification. Later the iTC modified the PP into a PP-Module for somewhat easier integration with the MDFPP (the primary target device at this time).
```

Comment:
"(the primary target device at this time)" :> "(the primary base PP at this time)"

Reply:
I have made this edit